### ISSUE TITLE
Background colour of textfield in survey screen is not updating for option 'other'. 

### ISSUE THREAD

**Issue by HithakshiKulal** [apptentive/apptentive-ios#263]
1. Present survey with choices along with **other** option. (Dashboard)
2. Customise UI to have dark backgound. Set textfield background to dark as well.  (app-end)
```
guard let style = Apptentive.sharedConnection().styleSheet as? ApptentiveStyleSheet else {
                return
        }
        style.setColor(
            .black,
            forStyle: ApptentiveStyleIdentifier.textInputBackground.rawValue
        )
```

Output: Background color isn't changing. 

Observation: textField in OtherCell (ApptentiveSurveyViewController) is never customised to change background color.

**Comment by CaseyApptentive** [comment]
Hi @HithakshiKulal ! Thanks for reaching out. Can you email us at support@apptentive.com with details on which app you're working on and if this is blocking you from release? That way, I can coordinate with your Customer Success Manager and prioritize the solution.

**Comment by HithakshiKulal** [comment]
@CaseyApptentive I found this bug when customising the UI (was exploring the SDK). This is not in our requirement as of now. so no worries!

**Comment by frankus** [comment]
This has been fixed as part of the 5.2.12 release.
